learning_resources_search/models.py

```diff
@@ -30,3 +34,28 @@ def __str__(self):
 
     class Meta:
         unique_together = (("source_type", "original_query"),)
+
+    def original_url_params(self):
+        ignore_params = ["endpoint"]
+        query = self.original_query
+        defined_params = {
+            key: query[key] for key in query if query[key] and key not in ignore_params
+        }
+        return unquote_plus(urlencode(defined_params, doseq=True))
```

Comment:
I think this should be
```suggestion
        return urlencode(defined_params, doseq=True)
```
because `Channel.search_filter` will urlencoded (after #1326)